com.unity.ml-agents/Runtime/Policies/BrainParameters.cs

```diff
@@ -39,10 +39,14 @@ public class BrainParameters
         [Range(1, 50)] public int numStackedVectorObservations = 1;
 
         /// <summary>
-        /// If continuous : The length of the float vector that represents the action.
-        /// If discrete : The number of possible values the action can take.
+        /// The length of the float vector that represents the action.
         /// </summary>
-        public int[] vectorActionSize = new[] {1};
+        public int vectorActionSize = 1;
```

Comment:
This will break all existing scenes, right?

Reply:
Yes